scripts/clean-large-folder.py

```diff
@@ -0,0 +1,31 @@
+import os
+import shutil
+import sys
+
+
+def compute_folder_size(folder):
+    total_size = 0
+    for dirpath, dirnames, filenames in os.walk(folder):
+        for f in filenames:
+            fp = os.path.join(dirpath, f)
+            # skip if it is symbolic link
+            if not os.path.islink(fp):
+                total_size += os.path.getsize(fp)
+
+    return total_size
+
+
+def clean_if_necessary(folder, threshold_size):
+    size = compute_folder_size(folder)
+    print(f"The size of {folder} is {size}, threshold is {threshold_size}")
+    if size > threshold_size:
+        print(f"Folder size exceeds the threshold, remove it")
```

Comment:
clean it

Reply:
Actually, the actions is just `rm`, so I guess the `remove` is suitable here.